cassandra/1.cql

```diff
@@ -12,7 +12,8 @@ CREATE TABLE devgym.task (
   year TIMESTAMP,
   type TEXT,
   timeuuid TIMEUUID,
-  task_description TEXT,
+  name TEXT,
```

Comment:
What if not every user would love to add this name? It is hard sometimes, isn't it?